schemas/profiles.v1.schema.json

```diff
@@ -0,0 +1,18 @@
+{
+  "$schema": "http://json-schema.org/draft-07/schema#",
+  "title": "profiles",
```

Comment:
I would call that `profile`, otherwise the generated class will have a plural name.